pom.xml

```diff
@@ -875,12 +875,16 @@
                 <plugin>
```

Comment:
I think you should first fix tests and then add violations. That way every commit has a chance to build. So please reorder commits.